src/background/services/wallet/index.ts

```diff
@@ -1386,47 +1380,13 @@ export default class WalletService extends GenericService {
     }
   };
 
-  async initSocket() {
-    const { apiHost, apiKey } = await this.exec('setting', 'getAPI');
-    const {hostname} = new URL(apiHost);
-    const is5pi = ['5pi.io', 'www.5pi.io'].includes(hostname);
-    const socket = bsocket.socket();
-    socket.on('connect', async () => {
-      try {
-        await socket.call(
-          'auth',
-          is5pi
-            ? '775f8ca39e1748a7b47ff16ad4b1b9ad'
-            : apiKey,
-        );
-        await socket.call('watch chain');
-      } catch (e) {
-        console.error(e);
-        return;
-      }
-    });
-
-    socket.on('error', (err: any) => {
-      console.error(err);
-    });
-
-    socket.on('disconnect', () => {
-      console.log('disconnected');
-    });
-
-    socket.bind('block connect', async (data: any) => {
-      setTimeout(async () => {
+  async initPoller() {
+    return setInterval(() => (async () => {
         await this.checkForRescan();
         const {hash, height, time} = await this.exec('node', 'getLatestBlock');
         await pushMessage(setInfo(hash, height, time));
         this.emit('newBlock', {hash, height, time});
-      }, 1000);
-
-    });
-
-    socket.connect(apiHost);
-
-    this.socket = socket;
+    })(), 5000);
```

Comment:
5s poller might be too often, considering handshake blocks are like 15 minutes per block.

how about make this a minute? i think it's already manually checking when users open/close popup